.github/workflows/deploy.yml

```diff
@@ -7,6 +7,44 @@ on:
   push:
     branches:
       - main
+    paths-ignore:
+      - '.cookiecutter/*'
+      - '.github/*'
```

Comment:
What about changes to `.github/workflows/` files?

Reply:
I don't think you'd want to trigger a deploy when a workflow file changes. For example if you change `ci.yml`, `codeql.yml`, `data_tasks.yml`, `redeploy.yml`, `report_refresh.yml`, `stale.yml` or `tasks.yml`: none of those go into the Docker image that gets built so it doesn't make sense to create and deploy a new image.

The only odd one out is `deploy.yml`. Perhaps deploys are failing and you fix a bug in `deploy.yml`. When merging that change you _might_ want it to run `deploy.yml` but I'm not sure you would always want to. I think you could trigger the workflow manually after merging your PR if you wanted to run it.

Reply:
Wondering if giving the user the ability to manually run any of the workflows might provide a solution.

[`workflow_dispatch`](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch) is being used on some, but not all of the workflows.
```
on:
  workflow_dispatch:
```

Reply:
Enabled running `ci.yml` manually: https://github.com/hypothesis/h/pull/7755/commits/f00d920de604e61202025bf64729d03815106613